keyboards/levinson/keymaps/atreus/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include "levinson.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```